src/scene/container/Container.ts

```diff
@@ -1395,15 +1400,15 @@ export class Container<C extends ContainerChild = ContainerChild> extends EventE
     }
 }
 
-Container.mixin(childrenHelperMixin);
-Container.mixin(getFastGlobalBoundsMixin);
-Container.mixin(toLocalGlobalMixin);
-Container.mixin(onRenderMixin);
-Container.mixin(measureMixin);
-Container.mixin(effectsMixin);
-Container.mixin(findMixin);
-Container.mixin(sortMixin);
-Container.mixin(cullingMixin);
-Container.mixin(cacheAsTextureMixin);
-Container.mixin(getGlobalMixin);
-Container.mixin(collectRenderablesMixin);
+extensions.mixin(Container, childrenHelperMixin);
```

Comment:
What if the args for mixin uses a rest args so you could do this (Target, ...sources[])? It would make it less verbose here.